ee-9/feature-pack/src/main/resources/feature_groups/domain-ha-profile.xml

```diff
@@ -11,5 +11,6 @@
     </feature-group>
 
     <feature-group name="basic-ha-profile"/>
+    <feature-group name="microprofile-opentracing-jaeger"/>
```

Comment:
that one might not be correct (see https://github.com/wildfly/wildfly/pull/13806#issuecomment-742650956)

Reply:
I think this is correct and the problem is in the 'wildfly' feature pack. In current master both 'ee-9/dist' and 'dist' include MP OT in their standard configs, while with this PR 'dist' no longer does.  As discussed starting at https://github.com/wildfly/wildfly/pull/13766#discussion_r540134348 we should decide separately about keeping MP subsystems in our standard configs other than standalone-microprofile[-ha].xml.